perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -3451,7 +3451,7 @@ public boolean removeVirtualAttribute(PerunSession sess, Resource resource, Grou
 	public boolean removeAttribute(PerunSession sess, User user, AttributeDefinition attribute) throws InternalErrorException {
 		try {
 			if (0 < jdbc.update("DELETE FROM user_attr_values WHERE attr_id=? AND user_id=?", attribute.getId(), user.getId())) {
-				log.info("Attribute (its value) was removed from user. Attribute={}, user={}", attribute, user);
+				log.debug("Attribute (its value) was removed from user. Attribute={}, user={}", attribute, user);
```

Comment:
"Attribute value was removed" is more concise than "Attribute (its value) was removed"